Issue 936 - standardize lesson submissions

acrymble opened the pull request:
This pull request closes #936 and adds a Proposal Form to the 'authors guidelines' page that encourages authors to use our new form.

@jenniferisasi can you please upload the Spanish lesson template form to the /assets/forms/ folder on branch Issue-936, and adapt the first sentence of /guia-para-autores so that people can find it? You said you updated the form but I don't have a copy so I can't do it myself.

The sentence on the English page reads: "If you have an idea for a new lesson, or have already written a tutorial that you think could be adapted for the Programming Historian, complete a lesson proposal form and contact..." and is linked on "proposal form". 

@mdlincoln I've uploaded the French form, but the French author's guidelines file isn't online yet, so I haven't been able to change it so that people can find this new form.

adamcrymble committed:
Create create.md

adamcrymble committed:
Add files via upload

adamcrymble committed:
Delete create.md

adamcrymble committed:
add proposal form link as per #936

mdlincoln commented:
@acrymble you will find the French guidelines file on the `french-staging` branch. If you can add it there now, then it'll publish when we eventually merge in that PR.

mdlincoln commented:
Update from 2018-11-29 meeting, remaining tasks are:

- [x] @jenniferisasi has a translation of the proposal form in Spanish. She will send to @acrymble to add to this PR. 
- [x] Link on the Spanish author guidelines to the Spanish proposal form

After that is done, we can merge this ticket. @mdlincoln will add the appropriate links for French himself.

jenniferisasi committed:
Add files via upload

Lesson proposal form in Spanish.

jenniferisasi committed:
Update guia-para-autores.md

Edited ## Traducir o proponer una lección nueva to add the new lesson proposal form to the paragraph.

mdlincoln committed:
point to actual file, not github source link

jenniferisasi commented:
I think we can close this issue?
